### PR TITLE
[FIX] body_html is not added by the mail composer

### PR DESCRIPTION
**Pull request by vrenaville**
* Purpose
The mail composer do not add body_html , we must add it in the fields that is required in the result

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

File: mail_inline_css/models/mail_template.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ class MailTemplate(models.Model):
 
     def generate_email(self, res_ids, fields=None):
         """Use `premailer` to convert styles to inline styles."""
+        if "body_html" not in fields:
+            fields.append('body_html')
         result = super().generate_email(res_ids, fields=fields)
         if isinstance(res_ids, int):
             result["body_html"] = self._premailer_apply_transform(result["body_html"])
```
Comment (leemannd) [inline mail_inline_css/models/mail_template.py:26]:
```suggestion
            result["body_html"] = self._premailer_apply_transform(
                result["body_html"])
```

Comment (leemannd) [reply mail_inline_css/models/mail_template.py:26]: Also on line 29

**Review by leemannd — COMMENTED** [review]
For the pre-commit hook linter

**Comment by leemannd** [comment]
@simahawk as former author, do you have time for a review?

File: mail_inline_css/models/mail_template.py [hunk 2]
```diff
@@ -19,6 +19,8 @@ class MailTemplate(models.Model):
 
     def generate_email(self, res_ids, fields=None):
         """Use `premailer` to convert styles to inline styles."""
+        if "body_html" not in fields:
```
Comment (leemannd) [inline mail_inline_css/models/mail_template.py:22]:
```suggestion
        if fields is not None and "body_html" not in fields:
```

**Review by leemannd — CHANGES_REQUESTED** [review]
To fix failing tests

**Comment by simahawk** [comment]
@i-vyshnevska you tackled this issue not a long ago. Can you check?

**Comment by leemannd** [comment]
@simahawk This is maybe what we where looking for? To be tested -> https://github.com/OCA/social/pull/549

**Comment by vrenaville** [comment]
@leemannd  I close it favor of https://github.com/OCA/social/pull/608

**Pull request closed without merging**